makefiles/testing.mak

```diff
@@ -17,6 +17,7 @@ UNITS=
 fuzz: TIMEOUT := $(shell timeout --version > /dev/null 2>&1 && echo 1 || echo 0)
 fuzz: $(CTAGS_TEST)
 	@ \
+	if [ $$(basename $(SHELL)) = zsh ]; then set -o SH_WORD_SPLIT; fi; \
```

Comment:
this way of detecting ZSH seem fragile to me, and I'd expect it to fail if ZSH is linked to /bin/sh (if that's even possible?).

I'd rather try to find something only ZSH defines, e.g. an environment variable like maybe `ZSH_SOMETHING` or alike.